app/views/protocol_importers/templates/_details.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+  <%= item[:details]&.any? ? "<b>#{t('protocol_importers.templates.warning.title') }</b>" : '' %>
```

Comment:
Did you mean `details.title` entry here instead of `warning.title`? Otherwise it's not used anywhere else.